tests/test_jobs_and_serve.py

```diff
@@ -233,63 +233,62 @@ def test_stop_spot_controller(self, _mock_cluster_state,
         assert not result.exception
         assert 'Cluster(s) not found' in result.output
 
         result = cli_runner.invoke(cli.autostop, ['-a'], input='n\n')
         assert isinstance(result.exception, SystemExit)
         assert 'Aborted' in result.output
 
-    def test_cancel_on_spot_controller(self, _mock_cluster_state,
-                                       _mock_spot_controller):
+    def test_cancel_on_job_controller(self, _mock_cluster_state,
+                                      _mock_job_controller):
         cli_runner = cli_testing.CliRunner()
-        result = cli_runner.invoke(cli.cancel,
-                                   [spot.SPOT_CONTROLLER_NAME, '-a'])
+        result = cli_runner.invoke(cli.cancel, [jobs.JOB_CONTROLLER_NAME, '-a'])
         assert result.exit_code == 1
-        assert 'Cancelling the spot controller\'s jobs is not allowed.' in str(
+        assert 'Cancelling the job controller\'s jobs is not allowed.' in str(
             result.output)
 
     @pytest.mark.timeout(60)
     def test_cancel(self, _mock_db_conn):
         cli_runner = cli_testing.CliRunner()
-        result = cli_runner.invoke(cli.spot_cancel, ['-a'])
+        result = cli_runner.invoke(cli.jobs_cancel, ['-a'])
         assert result.exit_code == 1
-        assert controller_utils.Controllers.SPOT_CONTROLLER.value.default_hint_if_non_existent in str(
+        assert controller_utils.Controllers.JOB_CONTROLLER.value.default_hint_if_non_existent in str(
             result.output), (result.exception, result.output, result.exc_info)
 
     @pytest.mark.timeout(60)
     def test_logs(self, _mock_db_conn):
         cli_runner = cli_testing.CliRunner()
-        result = cli_runner.invoke(cli.spot_logs, ['1'])
+        result = cli_runner.invoke(cli.jobs_logs, ['1'])
         assert result.exit_code == 1
-        assert controller_utils.Controllers.SPOT_CONTROLLER.value.default_hint_if_non_existent in str(
+        assert controller_utils.Controllers.JOB_CONTROLLER.value.default_hint_if_non_existent in str(
             result.output), (result.exception, result.output, result.exc_info)
 
     @pytest.mark.timeout(60)
     def test_queue(self, _mock_db_conn):
         cli_runner = cli_testing.CliRunner()
-        result = cli_runner.invoke(cli.spot_queue)
+        result = cli_runner.invoke(cli.jobs_queue)
         assert result.exit_code == 0
-        assert controller_utils.Controllers.SPOT_CONTROLLER.value.default_hint_if_non_existent in str(
+        assert controller_utils.Controllers.JOB_CONTROLLER.value.default_hint_if_non_existent in str(
             result.output), (result.exception, result.output, result.exc_info)
 
 
 class TestServeOperations:
     """Test operations for services."""
 
     @pytest.mark.timeout(60)
-    def test_down_spot_controller(self, _mock_cluster_state,
-                                  _mock_serve_controller, monkeypatch):
+    def test_down_job_controller(self, _mock_cluster_state,
```

Comment:
```suggestion
    def test_down_serve_controller(self, _mock_cluster_state,
```
seems like i accidentally copy-paste the function name loll